src/main/java/Homework2/Random_Chars_Table/RandomCharsTable.java

```diff
@@ -0,0 +1,84 @@
+package Homework2.Random_Chars_Table;
+
+import java.util.Scanner;
+
+public class RandomCharsTable {
+    int length = 0;
+    int height = 0;
+    String input = "";
+
+    static char getRandomChar(int min, int max) {
+        max -= min;
+        char randomCh = (char) ((int) (Math.random() * ++max) + min);
+        return randomCh;
+    }
+
+    public static char[][] getRandomCharsTable(int length, int height) {
+        char[][] randomCharTable = new char[length][height];
+        for (int i = 0; i < length; i++) {
+            for (int j = 0; j < height; j++) {
+                char ch = getRandomChar(65, 90);
+                randomCharTable[i][j] = ch;
+            }
+        }
+        return randomCharTable;
+    }
+
+    private boolean isValid(String input) {
+        return (input.matches("\\d+\\s\\d+\\sodd") || input.matches("\\d+\\s\\d+\\seven")
+                && height >= 0 && length >= 0);
```

Comment:
&& height >= 0 && length >= 0  ??????

verification is not correct.
height and length - set after verification. What is checked before? 
height and length cannot be 0. (replace - \\d+ ->>> [1-9]\\d* or [1-9][0-9]*)